packages/model-viewer/src/test/features/animation-spec.ts

```diff
@@ -227,9 +242,21 @@ suite('ModelViewerElementBase with AnimationMixin', () => {
         });
 
         test('plays the specified animation', () => {
-          expect(animationIsPlaying(element, element.availableAnimations[1]))
+          expect(animationWithIndexIsPlaying(element, 1))
               .to.be.true;
         });
+
+        suite('when playing an animation by name', () => {
+          setup(async () => {
+            element.animationName = element.availableAnimations[1];
+            await timePasses();
+          });
+  
+          test('fails to play the specified animation', () => {
+            expect(animationWithIndexIsPlaying(element, 1))
+                .to.be.false;
+          });
```

Comment:
I'm not sure if this type of test - testing that a known failure case is still failing - is in line with testing for model-viewer, feel free to remove again.

Reply:
Will you make that change or do you want me to? Sorry, wasn't quite clear

Reply:
Oh, sorry that wasn't clear. If you could, that'd be great 👍

Reply:
OK, pushed and clarified the wording a bit